pkg/network/status/status.go

```diff
@@ -356,10 +357,13 @@ func (m *Prober) processWorkItem() bool {
 			return dialContext(ctx, network, net.JoinHostPort(item.podIP, item.podPort))
 		}}
 
+	probeUrl := *item.url
```

Comment:
Strictly theoretically you can't copy the URL this way, `User       *Userinfo // username and password information` is not shallow copyable. If we can guarantee it will always be `nil`, it's fine though.

Reply:
I see, that's good to know. Is this preferable?
probeUrl := url.URL {
                     Scheme: *item.url.Scheme,
		    Host:   *item.url.Host,
		    Path:   probePath,
                      }

Reply:
shortest way to copy is
`newURL, _ := url.Parse(oldURL.String())`

Reply:
Thanks!